frontend/src/home/Home.tsx

```diff
@@ -185,8 +185,8 @@ interface ToastHomeProps {
 
 interface ReduxStoreHomeProps {
   schedule: DNDSchedule;
-  major?: Major;
-  planStr?: string;
+  major: string;
+  planStr: string;
```

Comment:
These should probably still be optional (and in other places as well)
This is causing some issues for majors that don't have options hooked up for coop cycles at the moment